client/apps/edit/client.coffee

```diff
@@ -27,10 +28,20 @@ async = require 'async'
       EditAdmin(article: @article, channel: channel)
       $('#edit-admin')[0]
     )
-    ReactDOM.render(
-      EditContent(article: @article, channel: channel)
-      $('#edit-content')[0]
-    )
+    if sd.EDIT_2
+      if channel.get('name') is 'Artsy Editorial'
+        # TODO - replace with better conditional (or channel.isEditorial),
+        # using name allows viewing the test channels in classic for interim.
+        @article.set('layout', 'standard')
```

Comment:
As long as you're in a staging environment, I don't see why you can't develop classic on a partner or team channel instead? The test channel is meant to emulate Editorial as much as possible which is why it's the only other channel with an `editorial` type.